.travis.yml

```diff
@@ -11,13 +11,12 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4
```

Comment:
And I found this problem.

The Travis CI build is failed because the dist `precise` and this dist does not have the `php-7.4` version.

Here are more detailed description about [this](https://docs.travis-ci.com/user/languages/php/).

If you want to support all of PHP versions (from `5.3` to `7.4`), you should define matrix setting to set different dist for different PHP versions.

Reply:
But it doesn't matter, this is not necessary once you decide to migrate to GitHub Action :).